tools/crunchbase.js

```diff
@@ -147,7 +147,7 @@ export async function fetchCrunchbaseEntries({cache, preferCache}) {
       var meAndParents = [result.data].concat(parents);
       var firstWithTicker = _.find( meAndParents, (org) => !!org.properties.stock_symbol );
       var firstWithFunding = _.find( meAndParents, (org) => !!org.properties.total_funding_usd );
-      if (firstWithTicker || c.ticker) {
+      if (!(c.ticker === null) && (firstWithTicker || c.ticker)) {
```

Comment:
I don't get it:
1) ticker === null
This is converted to `if (firstWithTicker)` in both cases
2) ticker !== null, i.e. it is present.
It is converted to `if (firstWithTicker || c.ticker)` in both cases

Reply:
This change helped to get around the error I was getting in https://app.netlify.com/sites/landscape/deploys/5af0b75d8df89449e3d0e8cd where I was setting `stock_ticker: null` in landscape.yaml but the build didn't recognize that meant it shouldn't use the ticker symbol from Crunchbase. I'm trying to skip this block if a null ticker is explicitly set.